plugins/woocommerce-admin/client/launch-your-store/hub/main-content/pages/launch-store-success/actions.tsx

```diff
@@ -12,8 +12,12 @@ import type { MainContentMachineContext } from '../../../main-content/xstate';
 
 export const assignCompleteSurvey = {
 	congratsScreen: ( { context }: { context: MainContentMachineContext } ) => {
-		dispatch( OPTIONS_STORE_NAME ).updateOptions( {
-			woocommerce_admin_launch_your_store_survey_completed: 'yes',
+		apiFetch( {
```

Comment:
It might throw errors in some cases. Do you think we should handle it? 

I'm not sure what happens if `apiFetch` throws an error but I think when using `dispatch( OPTIONS_STORE_NAME ).updateOptions`, it ignores the error.

Reply:
@chihsuan Updated in https://github.com/woocommerce/woocommerce/pull/47252/commits/a1ffaed5ee296be2ec06985061f8a1e2536090eb